### PR TITLE
chore/auto format lint

### PR DESCRIPTION
**Pull request by aidenstern**
On initial clone of the repo:
`npm install`
`npm run prepare`
And the pre-commit hooks with husky will be setup. Every commit will run eslint and prettier formatter on the staged files. This can be bypassed with git commit --no-verify however, so it would be best to add a github action that verifies the formatting and runs eslint again just to make sure.

### CODE REVIEW

**Review by thewilloftheshadow — APPROVED** [review]

**Review by thewilloftheshadow — CHANGES_REQUESTED** [review]
Could you add a section to the README.md that explains that you need to npm i and npm run prepare?

File: package.json [hunk 1]
```diff
@@ -4,7 +4,8 @@
   "description": "The only GroupMe API library that isn't a million years old.",
   "main": "dist/index.js",
   "scripts": {
-    "prepare": "npx tsc",
+    "build": "npx tsc",
+    "prepare": "husky install",
```
Comment (not-so-smart) [inline package.json:8]: This `prepare` script overwrites the previous one, breaking all of our current CI.

File: .prettierrc [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "trailingComma": "none"
```
Comment (not-so-smart) [inline .prettierrc:2]: I like trailing commas :(

**Review by not-so-smart — CHANGES_REQUESTED** [review]
1 potentially breaking issue (?) and 1 style preference

File: .prettierrc (hunk 2 above)
Comment (not-so-smart) [reply .prettierrc:2]: p.s. trailing commas are [a security matter](https://certitude.consulting/blog/en/invisible-backdoor/)

File: package.json (hunk 1 above)
Comment (thewilloftheshadow) [reply package.json:8]: I can change the CI, and let this be the new prepare/build scripts

Comment (not-so-smart) [reply package.json:8]: are you able to add commits to this PR? that would be desirable

Comment (thewilloftheshadow) [reply package.json:8]: I don't have write access to this repo

File: .prettierrc (hunk 2 above)
Comment (aidenstern) [reply .prettierrc:2]: Haha me too i was just adding this to test it out locally, ill leave it as an empty .prettierrc for now

**Comment by aidenstern** [comment]
Did you still want me to change the name of the `npm` scripts?

**Comment by not-so-smart** [comment]
> Did you still want me to change the name of the `npm` scripts?

I think this is up to @thewilloftheshadow

**Comment by thewilloftheshadow** [comment]
You can leave as is, and I'll rebase the other PR to use the new scripts

**Review by not-so-smart — APPROVED** [review]

**Review by thewilloftheshadow — APPROVED** [review]
